DataPackages/Goal/1/dimensionvalues.csv

```diff
@@ -1,4 +1,4 @@
-Dimension, Value, Description
+Dimension,Value,Description
```

Comment:
https://goodtables.io/github/roll/SDG/jobs/4

Here we have `Non-Matching Header`. Unfortunately for now `skipInitialSpace` is not fully supported by the goodtables.io chain (https://github.com/frictionlessdata/goodtables-py is already OK with it). So we had to remove white spaces. Just to clarify theoretically the initial version is also valid.